3-frames-and-windows/06-clickjacking/clickjacking-visible.view/facebook.html

```diff
@@ -3,7 +3,7 @@
 
 <body style="margin:10px;padding:10px">
 
-  <input type="button" onclick="alert('Like pressed on facebook.html!')" value="I LIKE IT !">
+  <input type="button" onclick="alert('¡"Me gusta" presionado en facebook.html!')" value="ME GUSTA !">
```

Comment:
Comillas anidadas... no funcionará 
Sin comillas, o podés intentar comillas latinas, o invertidas
```suggestion
  <input type="button" onclick="alert('¡ME GUSTA fue presionado en facebook.html!')" value="ME GUSTA !">
```